harness/Makefile

```diff
@@ -1,7 +1,10 @@
-py_bindings_dest=determined/common/api/bindings.py
+PY_BINDINGS_DEST=determined/common/api/bindings.py
+PY_SCHEMAS_DEST=determined/common/schemas/expconf/_gen.py
+SCHEMA_INPUTS = ../schemas/gen.py $(shell find ../schemas/expconf -name '*.json')
+GENERATED_COMMITTED := $(PY_SCHEMAS_DEST) $(PY_BINDINGS_DEST)
 
 .PHONY: build
-build: gen $(py_bindings_dest)
+build: gen
```

Comment:
question: is this removed dependency captured elsewhere? (other than maybe the higher level make file)

Reply:
`gen` depends on `GENERATED_COMMITTED`, which should by definition include the generated python bindings